src/PrestaShopBundle/Form/Admin/Improve/Design/Pages/CmsPageCategoryType.php

```diff
@@ -112,75 +125,83 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                             ),
                         ]),
                     ],
                 ],
             ])
             ->add('meta_description', TranslatableType::class, [
+                'label' => $this->trans('Meta description', 'Admin.Global'),
+                'help' => $invalidCharactersForNameLabel,
                 'required' => false,
                 'options' => [
                     'constraints' => [
                         new TypedRegex([
                             'type' => 'generic_name',
                         ]),
                         new Length([
-                            'max' => 512,
-                            'maxMessage' => $this->translator->trans(
+                            'max' => self::META_DESCRIPTION_MAX_LENGTH,
+                            'maxMessage' => $this->trans(
                                 'This field cannot be longer than %limit% characters',
-                                ['%limit%' => 512],
-                                'Admin.Notifications.Error'
+                                'Admin.Notifications.Error',
+                                ['%limit%' => self::META_DESCRIPTION_MAX_LENGTH]
                             ),
                         ]),
                     ],
                 ],
             ])
             ->add('meta_keywords', TranslatableType::class, [
+                'label' => $this->trans('Meta keywords', 'Admin.Global'),
+                'help' => $invalidCharactersForNameLabel,
                 'required' => false,
                 'options' => [
                     'constraints' => [
                         new TypedRegex([
                             'type' => 'generic_name',
                         ]),
                         new Length([
-                            'max' => 255,
-                            'maxMessage' => $this->translator->trans(
+                            'max' => self::META_KEYWORDS_MAX_LENGTH,
+                            'maxMessage' => $this->trans(
                                 'This field cannot be longer than %limit% characters',
-                                ['%limit%' => 255],
-                                'Admin.Notifications.Error'
+                                'Admin.Notifications.Error',
+                                ['%limit%' => self::META_KEYWORDS_MAX_LENGTH]
                             ),
                         ]),
                     ],
                     'attr' => [
-                        'placeholder' => $this->trans('Add tag', [], 'Admin.Actions'),
+                        'placeholder' => $this->trans('Add tag', 'Admin.Actions'),
                     ],
                 ],
             ])
             ->add('friendly_url', TranslatableType::class, [
+                'label' => $this->trans('Friendly URL', 'Admin.Global'),
+                'help' => $this->trans('Only letters and the minus (-) character are allowed.', 'Admin.Catalog.Help'),
```

Comment:
Regarding your comment in another PR (https://github.com/PrestaShop/PrestaShop/pull/22473#discussion_r547865863) should we change the help text here too for "Unless the 'Accented URL' option is enabled (in Shop parameters > Traffic & SEO), only letters, numbers, underscores (_), and hyphens (-) are allowed." ?

Reply:
I think that makes complete sense! Added it.